ui/pages/confirmations/components/signature-request-original/index.scss

```diff
@@ -122,8 +115,6 @@
   }
 
   &__row-value {
-    @include design-system.H6;
```

Comment:
Removing unneeded styles as this is covered by the `Text` component https://github.com/MetaMask/metamask-extension/pull/27076/files#diff-0b1e3dca811d174401e32d4e02e4f70ebf269526c26c58f788cb7c3f0ee28ce0R233